db/seeds/01_alta.rb

```diff
@@ -217,8 +217,8 @@
       weight_unit: Pet::WEIGHT_UNITS.sample,
       breed: Faker::Creature::Dog.breed,
```

Comment:
Completely inconsequential, but we might be cross breeding the cats with the dogs :sweat_smile:

Reply:
Haha you’re right. I’ll see if they have cat species